plugin-management-library/src/main/java/config/Config.java

```diff
@@ -0,0 +1,24 @@
+package config;
```

Comment:
Better package names will be needed for a library. E.g. "io.jenkins.tools.pluginmanager.lib" or so. Otherwise it will not be usable as Java API

Reply:
Ok, I can change that. And why do the short package names make it unusable?

Reply:
Well, it is rather about import logic in Java. If you have a short unqualified name in a lib, there is a much higher chance of collision. Also, it is more difficult to trace back the sources of code when looking in GitHub  and other locations

Reply:
Fixed in commit a78fb0dad53696ef3bd2c22a2c206e3c7caccdd1.